dokka-integration-tests/gradle/src/test/kotlin/AndroidComposeIT.kt

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package org.jetbrains.dokka.it.gradle
+
+import io.kotest.assertions.asClue
+import io.kotest.assertions.withClue
+import io.kotest.inspectors.shouldForAll
+import io.kotest.matchers.file.shouldBeAFile
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContain
+import io.kotest.matchers.string.shouldNotContain
+import org.gradle.testkit.runner.TaskOutcome.SUCCESS
+import org.gradle.testkit.runner.TaskOutcome.UP_TO_DATE
+import org.jetbrains.dokka.gradle.utils.*
+import org.jetbrains.dokka.gradle.utils.addArguments
+import org.jetbrains.dokka.gradle.utils.build
+import org.jetbrains.dokka.it.gradle.junit.*
+import kotlin.io.path.deleteRecursively
+
+/**
+ * Integration test for the `it-android-compose` project.
+ */
+@TestsDGPv2
+@TestsAndroidGradlePlugin
+@TestsCompose
+@WithGradleProperties(GradlePropertiesProvider.Android::class)
+class AndroidComposeIT {
+
+    @DokkaGradlePluginTest(sourceProjectName = "it-android-compose")
```

Comment:
is it possible to have `sourceProjectName` per class and not per method? As this is how most of our IT are configured

Reply:
I think that it should be or per-class or per-method, not both, otherwise yes, there are confusions. If KGP uses per-class, then let's leave it like this to be consistent.